app-crypt/pkcrack/pkcrack-1.2.2-r2.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+inherit toolchain-funcs
+
+DESCRIPTION="PkZip cipher breaker"
+HOMEPAGE="https://www.unix-ag.uni-kl.de/~conrad/krypto/pkcrack.html"
+SRC_URI="https://www.unix-ag.uni-kl.de/~conrad/krypto/pkcrack/${P}.tar.gz"
+
+LICENSE="pkcrack"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="!<app-text/html-xml-utils-5.3"
+BDEPEND="test? ( app-arch/zip[crypt] )"
+
+DOCS=(
+	doc/KNOWN_BUGS
+	doc/appnote.iz.txt
+	doc/README.W32
+	doc/pkzip.ps.gz
+	doc/CHANGES
+	doc/LIESMICH
+	doc/README.html
+	doc/README
+)
+
+PATCHES=(
+	"${FILESDIR}/${P}-build.patch"
+	"${FILESDIR}/${P}-gcc14-build.patch"
+)
+
+src_compile() {
+	cd src
+	emake CC="$(tc-getCC)" CFLAGS="${CFLAGS} ${LDFLAGS}" all
```

Comment:
emake -C src in both this & src_test lets you drop the `cd src`, but I wonder if S should just be set to include src, given every phase has to do the `cd`..

Reply:
I was thinking about it too, but didn't catch whether there's `src_configure` going on and I don't remember if `einstalldocs` worked either outside `${S}`